Vegeting/Screens/ChatRoom/Cell/InterestCollectionViewCell.swift

```diff
@@ -44,17 +46,24 @@ class InterestCollectionViewCell: UICollectionViewCell {
         addSubview(backgroundContentView)
         backgroundContentView.constraint(to: self)
         backgroundContentView.addSubview(interestLabel)
-        interestLabel.constraint(leading: backgroundContentView.leadingAnchor,
-                                                   trailing: backgroundContentView.trailingAnchor,
-                                                   centerY: self.centerYAnchor,
-                                                   padding: UIEdgeInsets(top: 0, left: 14, bottom: 0, right: 14))
+        interestLabel.constraint(top: backgroundContentView.topAnchor,
+                                 leading: backgroundContentView.leadingAnchor,
+                                 bottom: backgroundContentView.bottomAnchor,
+                                 trailing: backgroundContentView.trailingAnchor,
+                                 padding: UIEdgeInsets(top: 8, left: 14, bottom: 8, right: 14))
     }
     
     private func configureUI() {
         layer.masksToBounds = true
     }
-
+    
     func configure(with itemText: String) {
         interestLabel.text = itemText
     }
+    
+    func applySelectedState() {
+        backgroundContentView.backgroundColor = isSelectedCell ? .black : .systemGray6
+        interestLabel.textColor = isSelectedCell ? .white : .black
+        interestLabel.font = isSelectedCell ? .preferredFont(forTextStyle: .subheadline, compatibleWith: .init(legibilityWeight: .bold)) : .preferredFont(forTextStyle:.subheadline)
```

Comment:
p1;
위와 같은 방식으로 하게 됐을 때 이렇게 적용될 수 있습니다! ~~ 😁
```suggestion
      backgroundContentView.backgroundColor = isSelected ? .black : .systemGray6
        interestLabel.textColor = isSelected ? .white : .black
        interestLabel.font = isSelected ? .preferredFont(forTextStyle: .subheadline, compatibleWith: .init(legibilityWeight: .bold)) : .preferredFont(forTextStyle:.subheadline)
```

Reply:
https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Vegeting/pull/143/commits/f25642e9e854c1afa40d2ecf8461b3c42ed4e218#diff-621530c17340448afe600a661e652bcb33b97219af1cb0be72054c53c0783a11R69-R71
에서 반영하였습니다